HostOcean/docker-compose.yml

```diff
@@ -10,4 +10,31 @@ services:
       - data.sql
 
   data.sql:
-    image: microsoft/mssql-server-linux:latest
\ No newline at end of file
+    image: microsoft/mssql-server-linux:latest
+
+  elasticsearch:
+    image: docker.elastic.co/elasticsearch/elasticsearch:6.2.4
+    container_name: elasticsearch
+    ports:
+     - "9200:9200"
+    volumes:
+     - elasticsearch-data:/usr/share/elasticsearch/data
```

Comment:
Better provide this information at https://github.com/ArtemTereshkovich/HostOcean/blob/develop/HostOcean/docker-compose.override.yml